### DISCUSSION TITLE
(REQUEST) Zap gun stun multiplier

### DISCUSSION THREAD

**Discussion by TheComicRelief** [Malcolm-Q/LC-LateGameUpgrades#537] [Ideas]
I want the zap gun to have stun multiplier kinda like flashbangs do. For example, if you zap a giant for 5 seconds then after stopping zapping, it would get stunned for 6 seconds ( giants have stun multiplier of 1.2 so 5*1.2=6).

**Comment by WhiteSpike** [comment]
So like Aluminium Coils' increased stun but based on how long you keep it stun on? Just to make things clear.

**Comment by TheComicRelief** [comment]
yes
